app/components/transaction-list/transaction-list-item.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC } from 'react';
+import { useHover } from 'use-events';
+import { Box, Flex, Text } from '@blockstack/ui';
+import { Transaction } from '@blockstack/stacks-blockchain-sidecar-types';
+
+import { capitalize } from '../../utils/capitalize';
+import { getStxTransferDirection } from '../../utils/get-stx-transfer-direction';
+import { sumStxTxTotal } from '../../utils/sum-stx-tx-total';
+import { listHoverProps, EnableBefore } from './transaction-list-item-hover';
+import { TransactionIcon } from './transaction-icon';
+
+const dateOptions = {
+  year: 'numeric',
+  month: 'long',
+  day: 'numeric',
+  hour: 'numeric',
+  minute: 'numeric',
+  second: 'numeric',
+};
+
+interface TransactionListItemProps {
+  tx: Transaction;
+  address: string;
+}
+
+export const TransactionListItem: FC<TransactionListItemProps> = ({ tx, address }) => {
+  const [hovered, bind] = useHover();
+
+  const direction = getStxTransferDirection(address, tx);
+  const sumPrefix = direction === 'sent' ? '−' : '';
+  const memo =
+    tx.tx_type === 'token_transfer' &&
+    Buffer.from(tx.token_transfer.memo.replace('0x', ''), 'hex').toString('utf8');
```

Comment:
Is the memo always encoded in utf8? If they're created from the stacks-transactions library?

Reply:
Stole this from explorer. I'm not sure 🤷 , I guess it might not be, in which case it may show nonsense—is there a way to check it's encoding?

Reply:
I would just try it with a faucet TX - the memo is always "Faucet" I believe

Reply:
Have done, decodes as "Faucet"